temboardagent/plugins/dashboard/db.py

```diff
@@ -4,6 +4,7 @@
 import os
 import sqlite3
 from textwrap import dedent
+from temboardagent.httpd import JSONEncoder
```

Comment:
Perhaps it'd be cleaner to move the `JSONEncoder` class in the `tools` module rather than import `httpd` here? (Not sure.)

Reply:
I agree. Changed.